feat: High3DSound & Networked Entity Table

robert4049 opened the pull request:
…/tp-client_main.lua 	modified:   tp-advancedzombies/config.lua

This enables integration with High3DSounds; it's disabled by default as to not break existing deployments

robert4049 committed:
feat: High3DSound intergration 	modified:   tp-advancedzombies/client/tp-client_main.lua 	modified:   tp-advancedzombies/config.lua

robert4049 committed:
feat: networked entity table

robert4049 committed:
fix: CreatPed
	modified:   tp-advancedzombies/client/tp-client_main.lua

maxcamorra commented:
tested out and nothing works
1. on every if there is a "then" missing
2. the print with dump is not working
3. after all the changes and add the missing thens and remove the prints it shows me an error in SetPedSuffersCriticalHits, even though there nothing had been changed. The error for this is bad argument #1 to 'lower' (string expected, got nil)

robert4049 committed:
dry do not repeat yourself

robert4049 committed:
modified:   tp-advancedzombies/client/tp-client_main.lua

maxcamorra commented:
Still not working. Zombies are spawning but just standing around.

robert4049 committed:
fix: missing then on thread and createped
	modified:   tp-advancedzombies/client/tp-client_main.lua

robert4049 committed:
fix: another missing then

robert4049 committed:
finally fixed
	modified:   tp-advancedzombies/client/tp-client_main.lua
	modified:   tp-advancedzombies/server/tp-server_functions.lua

Manliketjb commented:
Zombies are spawning but just standing around.

HATE-dev commented:
network owerflow
